project_management/workshop_advert.md

```diff
@@ -0,0 +1,56 @@
+
+During this free workshop we will discuss reproducible computing environments, show examples of others’ projects in myBinder.org and help you learn how to prepare a binder-ready project. At the end of the workshop you will be able to take some of your own content (in a R or Jupyter notebook) and prepare it so that it can be used by others on myBinder.org.
+
+This workshop is for people who are:
+*	Interested in reproducibility, containers, Docker or continuous integration; 
+*	Already familiar with R Markdown or Jupyter notebooks;
+*	Happy to support others during practical sessions!
+
+### The Turing Way
+This workshop is being run by The Turing Way project team. The Turing Way will be a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do". Reproducible research is necessary to ensure that scientific work can be trusted. The goal is to ensure that all results can be independently verified and built upon in future work. Our interdisciplinary team includes researchers, research software engineers and librarians. This project is openly developed and any and all questions, comments and recommendations are welcome at our github repository: https://github.com/alan-turing-institute/the-turing-way.
+
+### Programme
+* 9.30 - 10.00: Coffee and Registration
+* 10.00 - 10.30: Introduction to the workshop and The Turing Way
+* 10.30 - 12.00: Reproducible computing environment and how Binder can help
+* 12.00 - 1.00: Lunch
+* 1.00 - 2.00: Zero to Binder
+* 2.00 - 2.30: Coffee
+* 2.30 - 3.30: Build your own Binder
+* 3.30 - 4.00: Demonstrating your Binder, general questions, feedback and close
+* 4.00 - 5.00: Optional hangout with instructors
+
+### Some example Binders:
+*	Office of National Statistics - a report on traffic flow as an early indicator for GDP
+*	A book on digital signal processing, or how computers understand music
+*	A collection of notebooks about neural networks
+*	Example of interactive code snippets in a normal webpage
+
+## Build a BinderHub
+
+Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. Researchers can be supported in doing this through myBinder.org which allows you to share your software with others in the form of a single clickable link, making it easier for users and removing the need for you to worry about supporting lots of different platforms. This is a huge step forward in terms of supporting reproducibility but myBinder.org requires all code and data to be made public, and computational power is limited. One solution to this is to host your own BinderHub locally allowing private code sharing and greater computational power. 
```

Comment:
Suggested edit: "and computational power and data storage are limited"

Reply:
I've made a different version of this sentence in my PR #154....trying to capture the same point: data and computational power is limited. Happy to adapt further to incorporate @sgibson91's suggestion if mine isn't clear.